cfpb/consumerfinance.gov
# Adding to Readme and adding filter to helm lint

Quick fix so that we only run the helm lint tests on actual changes to the helm files and not just every time. Also added documentation on how to run lint and template locally to the helm directory readme.

To double check that all yaml files are formatted to helm standard we employ the Helm cli tool [chart-testing](https://github.com/helm/chart-testing) specificly the lint [command](https://github.com/helm/chart-testing/blob/main/doc/ct_lint.md), and its related [Github Action](https://github.com/helm/chart-testing-action). Chart Testing or ct is meant to be used for linting and testing pull requests. It automatically detects charts changed against the target branch. It uses the ct.yaml file to know where the helm charts are located and which git branch to compare the changes. Additionally we use the github filter action to make sure the ct github action is only ran when there are changes to the helm directory yaml and tpl files.

---


## Additions

-Documentation on lint and template commands


## Changes

-Helm lint test should only run if there are changes to the yaml files in the helm directory instead of every PR


## How to test this PR

1.Look at the Readme 
2. Check that the helm lint test never ran as this doesn't change any helm yaml files


## Screenshots


## Notes and todos

-


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
